packages/grz-pydantic-models/src/grz_pydantic_models/submission/metadata/v1.py

```diff
@@ -1405,15 +1447,53 @@ def raise_if_index(message: str) -> None:
             )
 
 
-type Thresholds = dict[tuple[str, str, str], dict[str, Any]]
+ThresholdsDict = dict[
+    tuple[GenomicStudySubtype, LibraryType, SequenceSubtype, bool],
+    thresholds_model.Thresholds,
+]
+
 
+class _ThresholdEntry(StrictBaseModel):
+    """Threshold configuration for a particular sequencing setup."""
 
-def load_thresholds() -> Thresholds:
-    threshold_definitions = json.load(
+    model_config = ConfigDict(populate_by_name=True)
+
+    library_type: LibraryType = Field(alias="libraryType")
+    sequence_subtype: SequenceSubtype = Field(alias="sequenceSubtype")
+    genomic_study_subtype: GenomicStudySubtype = Field(alias="genomicStudySubtype")
+    thresholds: thresholds_model.Thresholds
+
+
+@cache
+def load_thresholds() -> ThresholdsDict:
+    threshold_definitions_json = json.load(
         files("grz_pydantic_models").joinpath("resources", "thresholds.json").open("r", encoding="utf-8")
     )
-    threshold_definitions = {
-        (d["genomicStudySubtype"], d["libraryType"], d["sequenceSubtype"]): d["thresholds"]
+    try:
+        threshold_definitions: list[_ThresholdEntry] = [
+            _ThresholdEntry.model_validate(item) for item in threshold_definitions_json
+        ]
+    except ValidationError as err:
+        raise RuntimeError("Invalid threshold definitions JSON") from err
+
+    threshold_definitions_dict = {
+        (d.genomic_study_subtype, d.library_type, d.sequence_subtype, False): d.thresholds
         for d in threshold_definitions
     }
-    return threshold_definitions
+    # add thresholds for oncomine panels
+    for d in threshold_definitions:
+        if d.library_type == LibraryType.panel:
+            thresholds = d.thresholds.model_copy(deep=True)
+            # we have different thresholds_model.percentBasesAboveQualityThreshold for oncomine panels:
+            # percentage of bases ≥ Q20 should be at least 70%
+            thresholds.percent_bases_above_quality_threshold = thresholds_model.PercentBasesAboveQualityThreshold(
+                qualityThreshold=20,
+                percentBasesAbove=70,
+            )
+
+            # add new entry for oncomine panels
+            threshold_definitions_dict[(d.genomic_study_subtype, d.library_type, d.sequence_subtype, False)] = (
```

Comment:
Shouldn't this key be `(d.genomic_study_subtype, d.library_type, d.sequence_subtype, True)`? Maybe an oncomine unit tests would be good.